tests/h/services/search_index/service_test.py

```diff
@@ -138,10 +134,6 @@ def test_it_does_nothing_if_the_annotation_is_deleted(
     def annotation(self, factories):
         return factories.Annotation.build()
 
-    @pytest.fixture(autouse=True)
-    def AnnotationSearchIndexPresenter(self, patch):
-        return patch("h.services.search_index.service.AnnotationSearchIndexPresenter")
```

Comment:
Same reasons as with `storage` above: this is a patch for a module-level import so it should be a module-scoped fixture

Reply:
Basically all imports yes, except in the rare case when you have an `import` inside a function or method. It's about namespaces, and `"AnnotationSearchIndexPresenter"` is a name in `service.py`'s top-level module namespace